lib/pecorino/adapters/postgres_adapter.rb

```diff
@@ -154,13 +159,36 @@ def set_block(key:, block_for:)
         blocked_until = GREATEST(EXCLUDED.blocked_until, t.blocked_until)
       RETURNING blocked_until
     SQL
-    model_class.connection.uncached { model_class.connection.select_value(block_set_query) }
+    @model_class.connection.uncached { @model_class.connection.select_value(block_set_query) }
   end
 
   def blocked_until(key:)
-    block_check_query = model_class.sanitize_sql_array([<<~SQL, key])
+    block_check_query = @model_class.sanitize_sql_array([<<~SQL, key])
       SELECT blocked_until FROM pecorino_blocks WHERE key = ? AND blocked_until >= clock_timestamp() LIMIT 1
     SQL
-    model_class.connection.uncached { model_class.connection.select_value(block_check_query) }
+    @model_class.connection.uncached { @model_class.connection.select_value(block_check_query) }
+  end
+
+  def prune
+    @model_class.connection.execute("DELETE FROM pecorino_blocks WHERE blocked_until < NOW()")
+    @model_class.connection.execute("DELETE FROM pecorino_leaky_buckets WHERE may_be_deleted_after < NOW()")
+  end
+
+  def create_tables(active_record_schema)
+    active_record_schema.create_table :pecorino_leaky_buckets, id: :uuid do |t|
```

Comment:
**suggestion:** would it be possible not to force :uuid as a primary key on users? Make it configurable?

Reply:
Yeah let's switch to whatever ID structure that is the app defalut 👍